packages/amplify-nodejs-function-runtime-provider/src/utils/legacyBuild.ts

```diff
@@ -39,19 +41,24 @@ function installDependencies(resourceDir: string) {
 }
 
 function runPackageManager(cwd: string, scriptName?: string) {
-  const isWindows = /^win/.test(process.platform);
-  const npm = isWindows ? 'npm.cmd' : 'npm';
-  const yarn = isWindows ? 'yarn.cmd' : 'yarn';
   const useYarn = fs.existsSync(`${cwd}/yarn.lock`);
-  const packageManager = useYarn ? yarn : npm;
+  const packageManager = useYarn ? 'yarn' : 'npm';
   const args = toPackageManagerArgs(useYarn, scriptName);
-  const childProcessResult = childProcess.spawnSync(packageManager, args, {
-    cwd,
-    stdio: 'pipe',
-    encoding: 'utf-8',
-  });
-  if (childProcessResult.status !== 0) {
-    throw new Error(childProcessResult.output.join());
+  try {
+    const { stdout, exitCode } = execa.commandSync(`${packageManager} ${args.join(' ')}`, {
```

Comment:
I think capturing stdout here will prevent it from being printed to the console (even with the `stdio: 'pipe'` option)